include/swift/Parse/Parser.h

```diff
@@ -1427,12 +1429,10 @@ class Parser {
   ///
   /// \returns true if an error occurred, false otherwise.
   bool parseClosureSignatureIfPresent(
-                                SmallVectorImpl<CaptureListEntry> &captureList,
-                                      ParameterList *&params,
-                                      SourceLoc &throwsLoc,
-                                      SourceLoc &arrowLoc,
-                                      TypeRepr *&explicitResultType,
-                                      SourceLoc &inLoc);
+      SourceRange &bracketRange, SmallVectorImpl<CaptureListEntry> &captureList,
+      VarDecl *&capturedSelfParamDecl, ParameterList *&params,
+      SourceLoc &throwsLoc, SourceLoc &arrowLoc, TypeRepr *&explicitResultType,
+      SourceLoc &inLoc);
```

Comment:
The re-indenting here was in response to @brentdax’s [comment](https://github.com/apple/swift/pull/23934#discussion_r313524806) up-thread! (Also, this code is *not* otherwise unchanged, since we’ve added the `bracketRange` and `capturedSelfParamDecl` parameters.)

Reply:
I believe that Brent actually did suggest one per line, but I corrected based on the output of `git-clang-format` which gave this style. I'll switch to one per line.